internal/services/notification/notification_test.go

```diff
@@ -496,16 +496,16 @@ func TestCommitStatusDelivery(t *testing.T) {
 	})
 }
 
-func createCommitStatus(t *testing.T, ctx context.Context, ns *NotificationService, n int) *types.CommitStatus {
+func createCommitStatus(t *testing.T, ctx context.Context, ns *NotificationService, index int, projectID string) *types.CommitStatus {
```

Comment:
why "index" ? it's used for RunCounter and for other unrelated things. Which is the primary use of it? "runCounter"? The other entries (commitSHA, Context etc...) are based on runCounter? If they are meaningless for the tests then document that their value is just a dumb value.